README.md

```diff
@@ -266,6 +277,7 @@ This section outlines all the flags you can set to customize the data prep pipel
 | `attention_boundary` | str | 'jsonl' | ['jsonl', 'prompt_completion_pair'] | The boundary to use when training with --article_attention flag. If you choose prompt_completion_pair tokens will only attend to tokens in the prompt_completion_pair. If you choose jsonl, then tokens will attend to all the prompt completion pairs in the jsonl |
 | `prompt_keyword` <span id="prompt_keyword"></span> | str | 'prompt' | | If your input json has a string keyword for prompt other than "prompt", place the keyword here. e.g Input_json: {"source": ... "target": ...} ->`prompt_keyword`='source'. |
 | `completion_keyword` | str | 'completion' | | If your input json has a string keyword for completion other than "completion", place the  keyword here. e.g Input_json: {"source": ... "target": ...} -> --completion_keyword='target'. |
+| `apply_chat_template` | bool | False | | Whether to tokenize the data using tokenizer.apply_chat_template, to add the chatML tags. The tokenizer must have apply_chat_template defined. |
```

Comment:
Few nits here:
1. Would be nice to add a little snippet on what `chatML` tags are here. I'm not sure users will all know what this means and I think it's too much info to throw into the smaller table above.
2. Not exactly sure what is meant by:
> The tokenizer must have apply_chat_template defined.

Does every new tokenizer need to implement the `apply_chat_template` base class or is there a supported list of tokenizers? A little more elaboration here would be super helpful.

3. Small formatting nit:

```suggestion
| `apply_chat_template` | bool | False | | Whether to tokenize the data using `tokenizer.apply_chat_template`, to add the chatML tags. The tokenizer must have apply_chat_template defined. |
```